templates/nginx.conf.j2

```diff
@@ -10,6 +10,36 @@ server {
 }
 {% endif -%}
 
+{% for redirector in app_nginx_redirect_names %}
+{% if redirector.ssl_enabled and redirector.ssl_only -%}
```

Comment:
@peakwinter I think you can remove this if scope here, duplicated from the bottom server instance.